doc/PluginTutorial.md

```diff
@@ -164,7 +164,7 @@ section:
 ```html
 	<head>
 		<!-- CSS -->
-		<link rel="stylesheet" href="../../css/roboto-condensed-min.css">
+		<link rel="stylesheet" href="../../css/roboto-condensed/css/roboto-condensed-min.css">
```

Comment:
The font-awesome folder structure was a lazy move. Let's make the css folder one layer deep ^_^